src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala

```diff
@@ -24,8 +27,21 @@ trait WorkspaceService extends HttpService with PerRequestCreator with FireCloud
   val routes: Route =
     pathPrefix("workspaces") {
       pathEnd {
-        passthrough(rawlsWorkspacesRoot, HttpMethods.GET, HttpMethods.POST)
-     } ~
+        passthrough(rawlsWorkspacesRoot, HttpMethods.GET) ~
+        post {
+          entity(as[WorkspaceCreate]) { createRequest => requestContext =>
+            val rawlsCreate = RawlsWorkspaceCreate(
+              namespace = createRequest.namespace,
+              name = createRequest.name,
+              attributes = createRequest.attributes.getOrElse(Map.empty),
+              realm = if (createRequest.isProtected.getOrElse(false))
+                        Some(Map("groupName" -> FireCloudConfig.Nih.rawlsGroupName))
+                      else None)
+            val extReq = Post(FireCloudConfig.Rawls.workspacesUrl, rawlsCreate)
+            externalHttpPerRequest(requestContext, extReq)
```

Comment:
I am fine with this approach - and it is consistent with how we've done things in orchestration elsewhere.

However, it seems more spray-like to write a new directive based on mapRequest (see: http://spray.io/documentation/1.2.3/spray-routing/basic-directives/mapRequest/#maprequest). I haven't tried it, but it looks like the mapRequest approach would allow us to keep the passthrough.

No action necessary for this PR. I'm commenting here to increase mindshare of this approach.